### PR TITLE
WINC-868: Add Kubernetes bump issue template

### PR DESCRIPTION
**Pull request by mansikulkarni96**
Adds option to create a K8s bump github issue

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
Example issue created from template on personal repo: https://github.com/mansikulkarni96/mansikulkarni96/issues/3

File: .github/ISSUE_TEMPLATE/kube-bump.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      options:
+        - label: OpenShift client and API dependencies are upgraded
+  - type: checkboxes
+    attributes:
+      label: Step 4 - Update kubelet and kube-proxy submodules in WMCO repository
+      options:
+        - label: kubelet and kube-proxy submodules are updated pointing to the latest commits including the rebase
+  - type: checkboxes
+    attributes:
+      label: Step 5 - Make sure SDN branch has to be created for kube-proxy submodule
```
Comment (saifshaikh48) [inline .github/ISSUE_TEMPLATE/kube-bump.yml:39]: How about `Engage SDN team to create new branch for kube-proxy submodule`? Gives more context on how to do this step

**Comment by saifshaikh48** [comment]
Also one question since I'm not as familiar with the process, is any of this work blocked by the [upstream rebase PR](https://github.com/openshift/kubernetes/pull/1360) landing? If so, consider calling it out

**Pull request closed without merging**